src/emu/debug/express.cpp

```diff
@@ -112,9 +112,9 @@ class integer_symbol_entry : public symbol_entry
 {
 public:
 	// construction/destruction
-	integer_symbol_entry(symbol_table &table, const char *name, symbol_table::read_write rw, u64 *ptr = nullptr);
-	integer_symbol_entry(symbol_table &table, const char *name, u64 constval);
-	integer_symbol_entry(symbol_table &table, const char *name, void *ref, symbol_table::getter_func getter, symbol_table::setter_func setter, const std::string &format);
+	integer_symbol_entry(symbol_table &table, std::string &&name, symbol_table::read_write rw, u64 *ptr = nullptr);
+	integer_symbol_entry(symbol_table &table, std::string &&name, u64 constval);
+	integer_symbol_entry(symbol_table &table, std::string &&name, void *ref, symbol_table::getter_func getter, symbol_table::setter_func setter, const std::string &format);
```

Comment:
If you're trying to cut down on temporaries, why not move the function object as well?